src/api/agents/adpters.ts

```diff
@@ -0,0 +1,61 @@
+type AgentConfig = {
+  integration_settings?: {
+    order_status_restriction?: {
+      phone_number: string;
+    };
+  };
+};
+
+export interface AgentsListResponse {
+  features: {
+    feature_uuid: string;
+    category: 'ACTIVE' | 'PASSIVE';
+    code: 'order_status' | 'abandoned_cart';
+    name: string;
+    description: string;
+    disclaimer: string;
+    documentation_url: string;
+    globals: string[];
+    config?: AgentConfig;
+    sectors: string[];
+    initial_flow: { uuid: string; name: string; }[];
+  }[]
+};
+
+function isInTest(config?: AgentConfig) {
+  return config?.integration_settings?.order_status_restriction?.phone_number
+    && config?.integration_settings?.order_status_restriction?.phone_number.length > 0 || false;
+}
```

Comment:
Suggestion to improve readability:

```suggestion
function isInTest(config?: AgentConfig) {
  const phoneNumber = config?.integration_settings?.order_status_restriction?.phone_number
  return phoneNumber?.length > 0 || false;
}
```

Reply:
great suggestion, thanks 🫂